packages/neuron-wallet/tests/services/sign-message.test.ts

```diff
@@ -94,7 +94,7 @@ describe(`SignMessage`, () => {
         walletMock.mockReturnValueOnce({ isHardware: () => true })
         const signMessage = jest.fn()
         hardWalletMock.mockReturnValueOnce({ signMessage })
-        await SignMessage.sign('walletId', signInfo.address, extendedKeyInfo.password, signInfo.message)
+        await SignMessage.sign('walletId', extendedKeyInfo.password, signInfo.message, signInfo.address)
```

Comment:
it is easier to understand when the parameters are refactored into an object, especially when the parameter count exceeds 3